spec/factories/users.rb

```diff
@@ -19,5 +19,9 @@
     trait :early_career_teacher do
       early_career_teacher_profile
     end
+
+    trait :mentor do
+      mentor_profile
```

Comment:
It turns out this creates a different user, then steals their mentor profile 😱 
Instead, I've been using `mentor_profile { build(:mentor_profile) }`. Slightly less pretty, but it doesn't mess up the cypress tests so much

Reply:
what xD WHAT??? WHAAAAAAAT?????

Fine, and I imagine we should do it for all of our traits?

Reply:
Yup 😢